assets/dev/js/editor/container/container.js

```diff
@@ -209,6 +209,57 @@ export default class Container extends ArgsObject {
 		return result;
 	}
 
+	/**
+	 * Function getUtilizedControls().
+	 *
+	 * Should return all controls that effecting the container.
+	 */
+	getUtilizedControls() {
```

Comment:
Maybe:

```suggestion
	getCustomizedControls() {
```

Reply:
Why?

Reply:
IF I understand you want to mark the controls that the user has been customized

Reply:
No, I want the controls that effects the container, if it's a default global, so the global control is utilized.

Reply:
Handled.